tests/python/test_with_dask.py

```diff
@@ -1285,6 +1286,82 @@ def test_dask_unsupported_features(client: "Client") -> None:
         )
 
 
+def test_parallel_submits(client: "Client") -> None:
+    """Test for running multiple train simultaneously from single clients."""
+    try:
```

Comment:
https://docs.pytest.org/en/stable/reference.html#pytest-importorskip

Reply:
Not sure if that's usable with sub-module.  Let me try.

Reply:
Tried it, that doesn't work with from import syntax:
``` python
pytest.mark.skipimport("distributed.MultiLock")
```